vllm/model_executor/tensorizer_loader.py

```diff
@@ -84,68 +83,71 @@ def __post_init__(self):
         self.s3_secret_access_key = (os.environ.get("S3_SECRET_ACCESS_KEY")
                                      or None)
         self.s3_endpoint = os.environ.get("S3_ENDPOINT_URL") or None
-
+        self.encryption_keyfile = _validate_keyfile_path(
+            self.encryption_keyfile,
+            self.file_obj
+        )
         self.stream_params = {
             "s3_access_key_id": self.s3_access_key_id,
             "s3_secret_access_key": self.s3_secret_access_key,
             "s3_endpoint": self.s3_endpoint,
-            "force_http": True,
+            "force_http": self.force_http
         }
 
         # Omitting self.dtype and self.device as this behaves weirdly
         self.deserializer_params = {
             "filter_func": self.filter_func,
-            "lazy_load": self.lazy_load,
-            "plaid_mode": bool(_is_vllm_model(file_uri=self.file_obj)),
-            "plaid_mode_buffers": self.plaid_mode_buffers,
             "verify_hash": self.verify_hash,
-            "encryption": self.deserializer_encryption_key,
-            # "dtype":self.dtype,
-            # "device":self.device,
+            "encryption": self.encryption_keyfile,
         }
+        if self.encryption_keyfile:
+            with stream_io.open_stream(
+                    self.encryption_keyfile,
+                    **self.stream_params,
+            ) as stream:
+                key = stream.read()
+                decryption_params = DecryptionParams.from_key(key)
+                self.deserializer_params['encryption'] = decryption_params
 
-    @staticmethod
     def add_cli_args(
-        parser: argparse.ArgumentParser, ) -> argparse.ArgumentParser:
+            parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         """Tensorizer CLI arguments"""
-        parser.add_argument(
-            "--serializer-encryption",
-            action="store_true",
-            help="An `EncryptionParams` object holding a password or key"
-            "to use for encryption. If None, no encryption will be used.",
-        )
-        parser.add_argument(
-            "--lazy-load",
-            action="store_true",
-            help="If True, tensors will be loaded and cached when keys are"
-            "accessed. If False, all tensors will be loaded into memory up"
-            "front.",
+
+        # Create the argument group
+        group = parser.add_argument_group(
+            'Tensorizer Options',
```

Comment:
Group names are lowercase by convention
```suggestion
            'tensorizer options',
```